pkg/apis/objectbucket.io/v1alpha1/objectbucketclaim_types.go

```diff
@@ -1,13 +1,37 @@
 package v1alpha1
 
 import (
+	"github.com/aws/aws-sdk-go-v2/service/s3"
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // ObjectBucketClaimSpec defines the desired state of ObjectBucketClaim
 type ObjectBucketClaimSpec struct {
+	// StorageClass names the StorageClass object representing the desired provisioner and parameters
 	StorageClass string
+	// BucketName (not recommended) the name of the bucket.  Caution!
+	// In-store bucket names may collide across namespaces.  If you define
+	// the name yourself, try to make it as unique as possible.
+	BucketName string `json:"bucketName,omityempty"`
+	// GenerateBucketName (recommended) a prefix for a bucket name to be
+	// followed by a hyphen and 5 random characters. Protects against
+	// in-store name collisions.
+	GeneratBucketName string `json:"generatBucketName,omitempty"`
```

Comment:
I'm a bit behind I guess. Looks like the current design does _not_ use the OBC's namespace as part of the generated bkt name?

Reply:
No, you're correct.  Not sure what I was thinking :P